mlflow/store/artifact/models_artifact_repo.py

```diff
@@ -25,37 +29,10 @@ def __init__(self, artifact_uri):
         uri = ModelsArtifactRepository.get_underlying_uri(artifact_uri)
         # TODO: it may be nice to fall back to the source URI explicitly here if for some reason
         #  we don't get a download URI here, or fail during the download itself.
-        self.repo = get_artifact_repository(uri)
-
-    @staticmethod
-    def _improper_model_uri_msg(uri):
-        return (
-            "Not a proper models:/ URI: %s. " % uri
-            + "Models URIs must be of the form 'models:/<model_name>/<version or stage>'."
-        )
-
-    @staticmethod
-    def _parse_uri(uri):
-        """
-        Returns (name, version, stage). Since a models:/ URI can only have one of {version, stage},
-        it will return (name, version, None) or (name, None, stage).
-        """
-        parsed = urllib.parse.urlparse(uri)
-        if parsed.scheme != "models":
-            raise MlflowException(ModelsArtifactRepository._improper_model_uri_msg(uri))
-
-        path = parsed.path
-        if not path.startswith("/") or len(path) <= 1:
-            raise MlflowException(ModelsArtifactRepository._improper_model_uri_msg(uri))
-        parts = path[1:].split("/")
-
-        if len(parts) != 2 or parts[0].strip() == "":
-            raise MlflowException(ModelsArtifactRepository._improper_model_uri_msg(uri))
-
-        if parts[1].isdigit():
-            return parts[0], int(parts[1]), None
+        if is_databricks_model_registry_artifacts_uri(uri):
```

Comment:
Yep, defined here : https://github.com/mlflow/mlflow/blob/master/mlflow/utils/uri.py#L240

Reply:
ah i see... i thought we were going to just do the databricks profile check like in `is_databricks_profile` instead of getting the underlying DBFS URI, since `DatabricksModelsArtifactRepository` doesn't care about the DBFS URI at all - why did we choose to check the DBFS path explicitly?